File: php_examples/generate_paraphrase_rewrite.php
<?php
$client = new Client();
$headers = [
  'Content-Type' => 'application/json'
];
$body = '{
  "template_name": "paraphrase",
  "prompt": {
    "original_sentence": "It was a great day at cortex HQ."
  },
  "temperature": 1.3,
  "word_count": 20,
  "n_gen": 4,
  "source_language": "en",
  "api_key": "YOUR_API_KEY"
}';
$request = new Request('POST', 'https://api.textcortex.com/hemingwai/generate_text_v2', $headers, $body);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();

File: generate_paraphrase_rewrite.py
import requests
import json

url = "https://api.textcortex.com/hemingwai/generate_text_v3"

payload = json.dumps({
  "template_name": "paraphrase",
  "prompt": {
    "original_sentence": "He also teaches architectural and urban design studios in several universities as an adjunct professor."
  },
  "temperature": 1,
  "token_count": 400,
  "n_gen": 2,
  "source_language": "en",
  "api_key": "YOUR_API_KEY"
})
headers = {
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)
